File: 20200424/code.py
def solution():
  n = int(input()) 
  arr = list(str(n))
  arr.sort(reverse=True)

  #10의 배수 아니면 -1
  if '0' not in arr: return -1
  #3의 배수 아니면 -1
  if n % 3 != 0: return -1

  answer = int(''.join(arr))

  return answer

print(solution())

File: 20200424/code.js
function solution(n) {
  let arr = n.toString().split('')
  arr.sort((a,b) => b - a);

  if (!arr.includes('0')) return -1;
  if (n % 3 != 0) return -1;

  return parseInt((arr.join("")));
}

console.log(solution(201));
